libsolidity/ast/Types.cpp

```diff
@@ -700,7 +698,16 @@ BoolResult FixedPointType::isImplicitlyConvertibleTo(Type const& _convertTo) con
 
 BoolResult FixedPointType::isExplicitlyConvertibleTo(Type const& _convertTo) const
 {
-	return _convertTo.category() == category() || _convertTo.category() == Category::Integer;
+	if (isImplicitlyConvertibleTo(_convertTo))
+		return true;
+	if (FixedBytesType const* convertTo = dynamic_cast<FixedBytesType const*>(&_convertTo))
```

Comment:
Not too sure about this one anymore.

Reply:
Yeah, I think we shouldn't do this now.

Reply:
Sorry, I am sure I want to have it now :)